custom/icds_reports/ucr/data_sources/ls_app_usage_forms.json

```diff
@@ -0,0 +1,159 @@
+          "http://openrosa.org/formdesigner/a884320881555582efdc2182b4af8f71429e345a",
+          "http://openrosa.org/formdesigner/e3a0e229d481f8ef6815b9dff283ec45202e66df",
+          "http://openrosa.org/formdesigner/58f155adce34955edd30f995d4a97acdcfdf267e",
+          "http://openrosa.org/formdesigner/BB672B78-DDA9-445A-AABF-7B775047270B"
+      ]
+    },
+    "validations": [
+      {
+        "name": "submitted_by_supervisor",
+        "error_message": "Form not submitted by supervisor",
+        "expression": {
+          "type": "boolean_expression",
+          "operator": "eq",
+          "expression": {
+            "location_id_expression": {
+              "type": "named",
+              "name": "user_location_id"
+            },
+            "type": "location_type_name"
+          },
+          "property_value": "supervisor"
+        }
+      }
+    ],
+    "configured_indicators": [
+      {
+        "display_name": "Supervisor ID",
+        "datatype": "string",
+        "type": "expression",
+        "transform": {},
+        "is_nullable": false,
+        "is_primary_key": true,
+        "column_id": "location_id",
+        "expression": {
+          "type": "named",
+          "name": "user_location_id"
+        }
+      },
+        {
+        "display_name": "State ID",
+        "transform": {},
+        "datatype": "string",
+        "expression": {
+          "location_id": {
+            "type": "named",
+            "name": "user_location_id"
+          },
+          "location_type": "state",
+          "location_property": "_id",
+          "type": "ancestor_location"
+        },
+        "is_primary_key": false,
+        "is_nullable": true,
+        "column_id": "state_id",
+        "type": "expression"
+      },
+      {
+        "display_name": null,
+        "transform": {},
+        "datatype": "datetime",
+        "expression": {
+          "expression": {
+            "datatype": "datetime",
+            "type": "property_path",
+            "property_path": [
+              "form",
+              "meta",
+              "timeEnd"
+            ]
+          },
+          "type": "root_doc"
+        },
+        "is_primary_key": false,
+        "is_nullable": true,
+        "column_id": "submitted_on",
+        "type": "expression"
+      }
+    ],
+    "named_expressions": {
+      "user_location_id": {
+        "type": "icds_user_location",
+        "user_id_expression": {
+          "type": "root_doc",
+          "expression": {
+            "type": "property_path",
+            "property_path": [
+              "form",
+              "meta",
+              "userID"
+            ]
+          }
+        }
+      }
+    },
+    "named_filters": {},
+    "engine_id": "icds-ucr-citus",
+    "disable_destructive_rebuild": true,
+    "sql_settings": {
+      "citus_config": {
+        "distribution_type": "hash",
+        "distribution_column": "location_id"
```

Comment:
This distributed ucr. Is anything special needs to do for this. Or not any other change is required.


Or you think this does not need to be distributed.
 @calellowitz

Reply:
Looks fine, and distributing it is a good idea. but i would change the column name to `supervisor_id` if for no other reason than consistency

Reply:
Moving distribution discussion to the same thread. From my other question:

> I think this should be supervisor_id. If we want to distribute this on Citus, using supervisor_id will ensure it is local to other data about the supervisor

@calellowitz isn't the change to `supervisor_id` needed to ensure data about a supervisor is located on the same shard? I recall you mentioning that when you use the same column name, citus distributes it in the same way. So the column name change is not just for consistency